fenix/app/src/main/java/org/mozilla/fenix/shopping/store/ReviewQualityCheckAction.kt

```diff
@@ -105,15 +110,55 @@ sealed interface ReviewQualityCheckAction : Action {
     /**
      * Triggered when the user clicks on learn more link on the opt in card.
      */
-    object OpenOnboardingLearnMoreLink : NavigationMiddlewareAction
+    object OpenOnboardingLearnMoreLink : NavigationMiddlewareAction, TelemetryAction
 
     /**
      * Triggered when the user clicks on terms and conditions link on the opt in card.
      */
-    object OpenOnboardingTermsLink : NavigationMiddlewareAction
+    object OpenOnboardingTermsLink : NavigationMiddlewareAction, TelemetryAction
 
     /**
      * Triggered when the user clicks on privacy policy link on the opt in card.
      */
-    object OpenOnboardingPrivacyPolicyLink : NavigationMiddlewareAction
+    object OpenOnboardingPrivacyPolicyLink : NavigationMiddlewareAction, TelemetryAction
+
+    /**
+     * Triggered when the bottom sheet is closed.
+     */
+    object BottomSheetClosed : TelemetryAction
+
+    /**
+     * Triggered when the bottom sheet is opened.
+     */
+    object BottomSheetDisplayed : TelemetryAction
+
+    /**
+     * Triggered when the user clicks on the "Not now" button from the contextual onboarding card.
+     */
+    object NotNowClicked : TelemetryAction
+
+    /**
+     * Triggered when the user expands the recent reviews card.
+     */
+    object ShowMoreRecentReviewsClicked : TelemetryAction
+
+    /**
+     * Triggered when the user expands the settings card.
+     */
+    object ExpandSettingsClicked : TelemetryAction
+
+    /**
+     * Triggered when the No analysis card is displayed to the user.
+     */
+    object NoAnalysisDisplayed : TelemetryAction
+
+    /**
+     * Triggered when the user clicks on the analyze button
+     */
+    object AnalyzeClicked : TelemetryAction
+
+    /**
+     * Triggered when the user reports a product is back in stock.
+     */
+    object ReactivateClicked : TelemetryAction
```

Comment:
`ReportProductBackInStock` seems like a better name even if the telemetry event is named like this